app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -453,16 +453,37 @@ function getConfigSelectors()
      * Returns the Skip Payment Method Step configuration.
      * @return string
      */
-    function isBoltOnlyPayment()
+    public function isBoltOnlyPayment()
     {
         return Mage::getStoreConfig('payment/boltpay/skip_payment');
     }
+    
+    /**
+     * Returns array of disabled customer group Id integers for the Bolt plugin
+     * @return array
+     */
+    public function getDisabledCustomerGroups()
+    {
+        return explode(',', Mage::getStoreConfig('payment/boltpay/bolt_disabled_customer_groups'));
+    }
+
+    /**
+     * Check if customer group ID in the disabled groups fo the Bolt Plugin
+     *
+     * @param integer $customerGroupId
+     * @return boolean
+     */
+    private function isCustomerGroupDisabled($customerGroupId)
+    {
+        $disabledCustomerGroups = $this->getDisabledCustomerGroups();
+        return in_array($customerGroupId, $disabledCustomerGroups);
+    }
 
     /**
      * Returns whether enable merchant scoped account.
      * @return string
      */
-    function isEnableMerchantScopedAccount()
+    public function isEnableMerchantScopedAccount()
```

Comment:
leave as private unless really needed to expose?

Reply:
@roopakv in the PHP by default functions are public. As @mccottry mentioned,  the code above is just explicitly declaring what is implicitly enforced.